remotespark/datawidgets/ipythondisplay.py

```diff
@@ -0,0 +1,11 @@
+from IPython.core.display import display, HTML
+
+
+class IpythonDisplay(object):
```

Comment:
It's not at the moment, but it's the type of methods that could very well be. My intention was to mock it when called from `with o:`... I will reconsider adding that unit test tomorrow, which means it will get mocked.